buildSrc/src/main/java/Dependencies.kt

```diff
@@ -28,9 +28,8 @@ private object Versions {
     const val androidx_core = "1.2.0-alpha01"
     const val androidx_transition = "1.1.0-rc01"
 
-    const val appservices_gradle_plugin = "0.4.4"
     const val mozilla_android_components = "0.53.0-SNAPSHOT"
-    const val mozilla_appservices = "0.27.0"
+    const val mozilla_appservices = "0.27.1"
```

Comment:
This got out of sync with the version used in android-components; if you like I could add some more logic to assert that we only see a single version number among any appservices dependencies.

Reply:
This got re-added with https://github.com/mozilla-mobile/fenix/pull/2282/files, and afaik only used for roboelectric tests. It'll be good to both comment on this fact here, and ensure that we don't diverge (if that's possible!)